[30032022 19:34:14] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[30032022 19:34:14] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[30032022 19:35:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:36:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:37:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:38:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:39:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:40:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:41:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:42:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:43:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:44:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:45:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:46:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:47:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:48:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:49:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:50:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:51:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:52:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:53:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:54:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:55:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:56:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:57:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:58:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 19:59:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:00:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:01:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:02:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:03:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:04:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:05:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:06:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:07:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:08:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:09:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:10:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:11:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:12:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:13:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:14:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:15:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:16:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:17:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:18:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:19:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:20:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:21:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:22:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:23:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:24:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:25:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:26:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:27:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:27:39] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[30032022 20:27:42] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[30032022 20:27:42] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[30032022 20:28:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:29:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:30:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:31:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:32:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:33:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:34:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:35:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:36:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:37:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:38:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:39:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:40:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:41:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:42:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:43:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:44:19] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[30032022 20:44:22] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[30032022 20:44:22] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[30032022 20:45:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:46:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:47:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:48:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:49:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:50:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:51:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:52:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:53:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:54:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:55:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:56:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:57:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:58:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 20:59:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:00:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:00:58] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 21:00:58] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 21:01:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:02:16] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 21:02:16] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 21:02:23] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:03:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:04:10] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 21:04:10] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 21:04:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:05:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:06:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:07:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:08:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:09:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:10:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:11:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:12:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:13:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:13:31] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[30032022 21:14:24] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:15:24] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:16:24] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:17:24] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:18:24] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:19:24] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:20:24] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:21:24] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:22:24] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:22:36] SMB /opt/dionaea/lib/dionaea/python/dionaea/smb/smb.py:127-warning: p.haslayer(Raw): b'PATH_REPLACE__?????\x00'
[30032022 21:22:36] SMB /opt/dionaea/lib/dionaea/python/dionaea/smb/smb.py:157-warning: p.haslayer(Raw): b'PATH_REPLACE__?????\x00'
[30032022 21:23:24] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:24:24] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:25:24] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:26:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:26:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:26:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 21:26:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[30032022 22:44:42] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[30032022 22:44:42] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
